Makefile

```diff
@@ -6,30 +6,28 @@ ifeq ($(NO_DOCKER), 1)
   DOCKER_CMD =
   IMAGE_BUILD_CMD = imagebuilder
   CGO_ENABLED = 1
+  GO = GO111MODULE=on GOFLAGS=-mod=vendor go
 else
-  DOCKER_CMD := docker run --rm -v "$(PWD)":/go/src/github.com/openshift/cluster-api-actuator-pkg:Z -w /go/src/github.com/openshift/cluster-api-actuator-pkg openshift/origin-release:golang-1.12
+  DOCKER_CMD := docker run --rm -v "$(PWD)":/cluster-api-actuator-pkg:Z -w /cluster-api-actuator-pkg -e GO111MODULE=on -e GOFLAGS=-mod=vendor openshift/origin-release:golang-1.12
   IMAGE_BUILD_CMD = docker build
+  GO = go
 endif
 
-.PHONY: depend
-depend:
-	dep version || go get -u github.com/golang/dep/cmd/dep
-	dep ensure
-
 .PHONY: depend-update
 depend-update:
-	dep ensure -update
+	go get -u
```

Comment:
This updates $THIS repo? Is that desired?

Reply:
Fixed in later commit.